packages/tocco-ui/src/EditableValue/utils.js

```diff
@@ -70,3 +70,65 @@ export const parseLocalePlaceholder = countryCode => {
 export const convertStringToNumber = stringValue => (
   !stringValue || isNaN(stringValue) ? null : parseFloat(stringValue)
 )
+
+/*
+ * Convert two numbers as hours and minutes to milliseconds
+ */
+export const calculateMilliseconds = (hours, minutes) => {
+  if (!hours && !minutes) {
+    return null
+  }
+  const hoursMilliseconds = (hours || 0) * 60 * 60000
+  const minutesMilliseconds = (minutes || 0) * 60000
+  return hoursMilliseconds + minutesMilliseconds
+}
+
+/*
+ * Return true if the value is null or undefined
+ */
+export const isNullOrUndefined = value => value === null || value === undefined
+
+/*
+ * Return true if the value is null or undefined
+ */
+export const millisecondsToDuration = milliSeconds => {
+  if (!milliSeconds && milliSeconds !== 0) {
+    return {hoursOfDay: 0, minutesOfHour: 0}
+  }
+
+  const hoursOfDay = Math.floor(milliSeconds / (60 * 60 * 1000))
+  const minutesOfHour = Math.floor((milliSeconds - (hoursOfDay * (60 * 60 * 1000))) / (60 * 1000))
+  return {hoursOfDay, minutesOfHour}
+}
+
+/*
+ * Convert a timeString of type HH:MM to time number object
+ */
+export const stringToDuration = timeString => {
+  if (!timeString && timeString !== '') {
+    return {hourOfDay: 0, minuteOfHour: 0}
+  }
+  const hourOfDay = parseInt(timeString.split(':')[0])
+  const minuteOfHour = parseInt(timeString.split(':')[1])
+  return {hourOfDay, minuteOfHour}
+}
+
+/*
+ * Convert hour and minute numbers to time string of type HH:MM
+ */
+export const numbersToTimeFormat = (hours, minutes) => {
+  if (!hours && !minutes) {
+    return '--:--'
+  }
+  const hourString = hours ? hours.toString() : '00'
+  const minuteString = minutes ? minutes.toString() : '00'
+  const timeArray = [hourString, minuteString]
```

Comment:
why do you push this into an array? you know that there are always 2 values (hours and minutes)